.github/workflows/maven-build.yml

```diff
@@ -4,7 +4,7 @@
 name: Java CI with Maven
 
 env:
-  PROFILES: frankframework,codecoverage,ci
+  PROFILES: frankframework,codecoverage,ci,database-drivers
```

Comment:
is dit echt nodig?

Reply:
It is needed for running spring-boot. It might also be possible to include h2 as test dependency.

Reply:
Like how we use iaf-example (with a h2 dependency)? I'd prefer as little overhead as possible, and that also makes running it locally a lot simpler?